frontend/package.json

```diff
@@ -8,11 +8,10 @@
     "build": "vue-cli-service build",
     "dev": "yarn run serve",
     "analyse-bundle": "yarn build && webpack-bundle-analyzer dist/webpack.stats.json",
-    "lint": "eslint --max-warnings=0 --ext .js,.vue .",
+    "lint": "eslint --max-warnings=0 --ext .js,.vue,.json .",
     "stylelint": "stylelint --max-warnings=0 '**/*.{scss,vue}'",
-    "i18n:report": "vue-cli-service i18n:report --src './src/**/*.?(js|vue)' --locales './src/locales/**/*.json'",
     "test": "TZ=UTC jest --coverage",
-    "locales": "scripts/missing-keys.sh && scripts/sort.sh"
+    "locales": "scripts/sort.sh"
```

Comment:
Why did you remove the missing keys here?

Reply:
This tests that the keys in de and en locales are exactly the same

Reply:
I use `'@intlify/vue-i18n/no-missing-keys-in-other-locales': 'error',`